recipe/meta.yaml

```diff
@@ -71,7 +75,7 @@ requirements:
     - libopenblas
```

Comment:
This is unnecessary and wrong.

Reply:
Should that be `blas` (or something else) or do I need to remove the line entirely?

Reply:
Remove entirely. Lines above this line are enough.

Reply:
Ok, thanks!